jina/hubapi/docker.py

```diff
@@ -91,8 +91,9 @@ def pull(self):
             image = self._client.images.pull(self.args.name)
         if isinstance(image, list):
             image = image[0]
+        image_tag = image.tags[0] if len(image.tags) > 0 else ""
```

Comment:
please do not use `len()` to check if list empty

```suggestion
        image_tag = image.tags[0] if image.tags else ''
```

Reply:
fixed